go.mod

```diff
@@ -1,79 +1,55 @@
 module github.com/openyurtio/openyurt
 
-go 1.13
+go 1.16
 
 require (
-	github.com/Microsoft/go-winio v0.4.14
-	github.com/NYTimes/gziphandler v1.1.1 // indirect
-	github.com/aliyun/alibaba-cloud-sdk-go v1.61.355
-	github.com/daviddengcn/go-colortext v0.0.0-20160507010035-511bcaf42ccd
-	github.com/docker/docker v17.12.0-ce-rc1.0.20200531234253-77e06fda0c94+incompatible // indirect
+	github.com/Microsoft/go-winio v0.4.11
+	github.com/aliyun/alibaba-cloud-sdk-go v1.61.579
+	github.com/daviddengcn/go-colortext v1.0.0
 	github.com/emicklei/go-restful v2.12.0+incompatible // indirect
-	github.com/evanphx/json-patch v4.5.0+incompatible
-	github.com/go-openapi/spec v0.19.8 // indirect
-	github.com/google/uuid v1.1.1
+	github.com/evanphx/json-patch v4.11.0+incompatible
+	github.com/fsnotify/fsnotify v1.4.10-0.20200417215612-7f4cf4dd2b52 // indirect
+	github.com/google/uuid v1.1.2
 	github.com/gorilla/mux v1.7.4
 	github.com/lithammer/dedent v1.1.0
-	github.com/onsi/ginkgo v1.13.0
-	github.com/onsi/gomega v1.10.1
-	github.com/opencontainers/selinux v1.3.1-0.20190929122143-5215b1806f52
+	github.com/onsi/ginkgo v1.14.1
+	github.com/onsi/gomega v1.10.2
+	github.com/opencontainers/selinux v1.10.0
 	github.com/openyurtio/yurt-app-manager-api v0.18.8
-	github.com/pkg/errors v0.8.1
-	github.com/prometheus/client_golang v1.7.1
-	github.com/spf13/cobra v1.0.0
+	github.com/pkg/errors v0.9.1
+	github.com/prometheus/client_golang v1.11.0
+	github.com/spf13/cobra v1.1.3
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.7.0
-	github.com/vishvananda/netlink v1.0.0
-	golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d // indirect
-	golang.org/x/sys v0.0.0-20200519105757-fe76b779f299
-	golang.org/x/time v0.0.0-20200416051211-89c76fbcd5d1 // indirect
-	google.golang.org/grpc v1.27.0
+	github.com/vishvananda/netlink v1.1.1-0.20200603190939-5a869a71f0cb
+	golang.org/x/sys v0.0.0-20210616094352-59db8d763f22
+	google.golang.org/grpc v1.40.0
 	gopkg.in/cheggaaa/pb.v1 v1.0.25
-	gopkg.in/square/go-jose.v2 v2.5.1 // indirect
-	k8s.io/api v0.19.7
-	k8s.io/apimachinery v0.19.7
-	k8s.io/apiserver v0.18.8
-	k8s.io/cli-runtime v0.18.8
-	k8s.io/client-go v0.19.2
-	k8s.io/cluster-bootstrap v0.0.0
-	k8s.io/component-base v0.18.8
+	k8s.io/api v0.23.1
+	k8s.io/apimachinery v0.22.3
+	k8s.io/apiserver v0.22.3
+	k8s.io/client-go v0.22.3
+	k8s.io/cluster-bootstrap v0.22.3
+	k8s.io/component-base v0.22.3
+	k8s.io/component-helpers v0.22.3
+	k8s.io/controller-manager v0.22.3
 	k8s.io/klog/v2 v2.30.0
-	k8s.io/kube-controller-manager v0.0.0
-	k8s.io/kubectl v0.0.0
-	k8s.io/kubelet v0.0.0
-	k8s.io/kubernetes v1.18.8
-	k8s.io/system-validators v1.0.4
-	k8s.io/utils v0.0.0-20210819203725-bdf08cb9a70a
+	k8s.io/kube-controller-manager v0.22.3
+	k8s.io/kubelet v0.22.3
+	k8s.io/system-validators v1.6.0
+	k8s.io/utils v0.0.0-20210930125809-cb0fa318a74b
```

Comment:
In require block, how about directly using `k8s.io/api v0.22.3`, `k8s.io/klog/v2 v2.9.0` ?

Reply:
fixed

Reply:
@rambohe-ch BTW, cloud I know the reason why we need to require and replace the package with the same version, such as :
```
require (
  ...
  k8s.io/apimachinery v0.22.3
  ...
)
replace (
  ...
  k8s.io/apimachinery => k8s.io/apimachinery v0.22.3
  ...
)
```
In my mind, we just need to do so for `sigs.k8s.io/apiserver-network-proxy v0.0.15`, because we forked it and modified for openyurt.
Thanks.